packages/table/src/TableActionMenu.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, DropdownToggle, DropdownMenu } from 'reactstrap';
+
+const TableActionMenu = ({ id, children }) => {
+  const [isToggled, toggle] = useState(false);
```

Comment:
I think we should swap the way the setter functions are named and update the name of the state variable
```
const [isOpen, setIsOpen] = useState(false)
const toggle = () => setIsOpen(!isOpen)
```